python/mxnet/ndarray/numpy/random.py

```diff
@@ -279,6 +280,44 @@ def multinomial(n, pvals, size=None):
         return _npi.multinomial(n=n, pvals=pvals, size=size)
 
 
+def rayleigh(scale=1.0, size=None, ctx=None, out=None):
+    r"""Draw samples from a Rayleigh distribution.
+
+    The :math:`\chi` and Weibull distributions are generalizations of the
+    Rayleigh.
+
+    Parameters
+    ----------
+    scale : float or array_like of floats, optional
```

Comment:
not `array_like`, we should only support either `mx.numpy.ndarray` or a float scalar.